cmd/snap-bootstrap/cmd_initramfs_mounts.go

```diff
@@ -335,6 +335,12 @@ func doInstall(mst *initramfsMountsState, model *asserts.Model, sysSnaps map[sna
 	}
 
 	if trustedInstallObserver != nil {
+		if err := trustedInstallObserver.ObserveExistingTrustedRecoveryAssets(boot.InitramfsUbuntuSeedDir); err != nil {
```

Comment:
does this change needs a new test? also does this merit a comment?

Reply:
I have mocked TrustedInstallObserver in order to test it.